server/src/internalClusterTest/java/org/elasticsearch/snapshots/DedicatedClusterSnapshotRestoreIT.java

```diff
@@ -1007,6 +1013,67 @@ public void testPartialSnapshotsDoNotRecordDeletedShardFailures() throws Excepti
         assertThat(snapshotInfo.shardFailures(), empty());
     }
 
+    public void testDeleteIndexDuringSnapshot() throws Exception {
+        final String indexName = "test-idx";
+        assertAcked(prepareCreate(indexName, 1, indexSettingsNoReplicas(1)));
+        ensureGreen();
+        indexRandomDocs(indexName, 100);
+
+        final String repoName = "test-repo";
+        createRepository(repoName, "mock",
+                Settings.builder().put("location", randomRepoPath()).put("random", randomAlphaOfLength(10))
```

Comment:
We don't use the extra settings here:
```suggestion
                Settings.builder().put("location", randomRepoPath()));
```